LICENSE-3rdparty.csv

```diff
@@ -75,3 +75,4 @@ dev,webpack,MIT,Copyright JS Foundation and other contributors
 dev,webpack-cli,MIT,Copyright JS Foundation and other contributors
 dev,webpack-dev-middleware,MIT,Copyright JS Foundation and other contributors
 dev,webpack-webextension-plugin,MIT,Copyright 2018 Henrik Wenz (handtrix@gmail.com)
+dev,node-fetch,MIT,Copyright 2016-2020 Node Fetch Team
```

Comment:
```suggestion
dev,node-fetch,MIT,Copyright 2016-2020 Node Fetch Team

```
can you also update you editor conf to not remove new line at the end of files?
(second time in this PR)